cpp/Dockerfile.ubuntu-bionic

```diff
@@ -92,14 +95,11 @@ RUN apt-get update -y -q && \
 ENV CMAKE_ARGS="-DThrift_SOURCE=BUNDLED \
 -DFlatbuffers_SOURCE=BUNDLED \
 -DGTest_SOURCE=BUNDLED \
--DORC_SOURCE=BUNDLED \
--Dc-ares_SOURCE=BUNDLED \
--DgRPC_SOURCE=BUNDLED \
--DProtobuf_SOURCE=BUNDLED ${CMAKE_ARGS}"
```

Comment:
Can we remove `${CMAKE_ARGS}` too?

Reply:
I'll leave this one for now because it seems used by other scripts/images, e.g.

```
r/Dockerfile.sanitizer
61:ENV CMAKE_ARGS="-DThrift_SOURCE=BUNDLED \
66:-DProtobuf_SOURCE=BUNDLED ${CMAKE_ARGS}"
```